go.mod

```diff
@@ -285,11 +284,8 @@ replace (
 	// Use cosmos keyring
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
 
-	// dragonberry
-	github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8.0
```

Comment:
Per the [SDK's release notes](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.45.12), it's advised to also bump `github.com/confio/ics23/go v0.9.0`...while I see this has been indirectly required, given the nature of the version bump and the SDK's recommendation we should explicitly require this.

Reply:
SDK says:
# Can be deleted from go.mod
replace github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8.0
Instead, github.com/confio/ics23/go must be bumped to v0.9.0.

Which is what I've done.

Reply:
It's being required indirectly [here](https://github.com/cosmos/gaia/pull/2088/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R72)...we should explicitly require this version.